generators/app/templates/src/index.js

```diff
@@ -0,0 +1,51 @@
+export default {
+    // Multiple browsers support
+    isMultiBrowser: <%= isMultiBrowser %>,
+
+
+    // Required - must be implemented
+    // Browser control
+    async openBrowser (/* id, browserName, startPage */) {
+        throw new Error('Not implemented!');
+    },
+
+    async closeBrowser (/* id, pageInfo */) {
```

Comment:
`pageInfo` is really `pageInfo`, I swear :), check it here: https://github.com/DevExpress/testcafe/blob/master/src/client/driver/command-executors/prepare-browser-manipulation.js#L36

Its `title` property is used to identify browser windows for local browsers. Also, some time ago we used `url` to track IE windows, so IMHO keeping it in `pageInfo` is not a bad idea. `width` and `height` properties are used to calculate corrections when resizing to the specified client area size.